src/sinks/kafka.rs

```diff
@@ -332,7 +363,8 @@ impl Sink<Event> for KafkaSink {
 }
 
 async fn healthcheck(config: KafkaSinkConfig) -> crate::Result<()> {
-    let client = config.to_rdkafka().unwrap();
+    trace!("Healthcheck started.");
```

Comment:
Are these logs common? Is it strange to have them for the `kafka` sink and nothing else?